odl/operator/default_ops.py

```diff
@@ -182,7 +182,7 @@ class LinCombOperator(Operator):
         LinCombOperator(a, b)([x, y]) == a * x + b * y
     """
 
-    def __init__(self, space, a, b):
+    def __init__(self, space, a, b, rangeType=None):
```

Comment:
Oh, that wasn't meant to be pushed... I used it for a mapping-oparator from R2 to C...